homeassistant/components/binary_sensor/sleepiq.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
I've based this on existing components. I'm not a fan of it myself, but most of them just say they are support for this and that. Examples:

* https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/ecobee.py
  * https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/ecobee.py
* https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/fitbit.py

Anything in particular I can answer?